src/Controllers/Authentication/login.controller.js

```diff
@@ -3,51 +3,83 @@
 import pool from "../../Database/database";
 import { verify } from "../../middleware/hash-password";
 import { signToken } from "../../middleware/jwt";
+// import jwt from "jsonwebtoken";
+//import Token from "../../models/token";
 
 export const login = async (req, res) => {
-	const { password, email } = req.body;
+  const { password, email } = req.body;
 
-	let user = await pool.query(
-		`SELECT * FROM public."Users" WHERE email LIKE '%${email}%' LIMIT 1`
-	);
+  let user = await pool.query(
+    `SELECT * FROM public."Users" WHERE email LIKE '%${email}%' LIMIT 1`
+  );
 
-	if (email == "admin@gmail.com" && password == "password") {
-		const userData = {
-			name: "Admin User",
-			role: "admin",
-			email: email,
-			createdAt: new Date(),
-			id: "1",
-		};
-		const token = signToken(JSON.stringify(userData));
-		return res.status(200).json({
-			success: true,
-			message: "successfully Admin logged in",
-			token,
-		});
-	} else {
-		if (!user.rowCount) {
-			return res
-				.status(401)
-				.send({ success: false, message: `Invalid email or password` });
-		}
-		const isPasswordValid = await verify(password, user.rows[0].password);
-		if (!isPasswordValid)
-			return res
-				.status(401)
-				.json({ success: false, message: "Invalid email or password" });
-		const userData = {
-			name: user.rows[0].name,
-			role: user.rows[0].role,
-			email: user.rows[0].email,
-			createdAt: user.rows[0].createdAt,
-			id: user.rows[0].id,
-		};
-		const token = signToken(JSON.stringify(userData));
-		return res.status(200).json({
-			success: true,
-			message: "successfully logged in",
-			token,
-		});
-	}
+  if (email == "admin@gmail.com" && password == "password") {
+    const userData = {
+      name: "Admin User",
+      role: "admin",
+      email: email,
+      createdAt: new Date(),
+      id: 1,
+    };
+     const token = signToken(JSON.stringify(userData));
+    // const TokenInDatabase =await Token.create({token_content:token, userId:userData.id});
```

Comment:
also remember to remove the unused or commented code block

Reply:
Yeah Done!!!!, Thanks